CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Remove broken links in addon and extension dir (#172)
 
 ### Changed 
+- Updated dependencies. Now oldest supported VS Code version is `1.28.0`. ([#137](https://github.com/JacquesLucke/blender_vscode/pull/147))
```

Comment:
When changing the minimum supported version, it's typically good to mention when that version was released.